docs/maui/views/MediaElement.md

```diff
@@ -152,6 +212,30 @@ By default, the media that is defined by the `Source` property doesn't immediate
 
 Platform provided media playback controls are enabled by default, and can be disabled by setting the `ShouldShowPlaybackControls` property to `false`.
 
+### Using MetaData
+
+A `MediaElement` can use metadata for `MediaElement.MetaDataTitle`, `MediaElement.MetaDataArtist` and `MediaElement.MetaDataArtworkUrl`. You can set
+the title or artist to show what is currently playing on lockscreen controls for Windows, Mac Catalyst, iOS, and Android. You can set a local or 
+remote URL with artwork for the lockscreen. It should be at least 1080P for best quality to be displayed. It must be a URL and be either `.jpg` or
+ `.png` 
+
+```xaml
+<toolkit:MediaElement 
+    MetaDataTitle="Title"
+    MetaDataArtist="Artist"
+    MetaDataArtworkUrl="http://www.myownpersonaldomain.com/image.jpg" />
+```
+
+```csharp
+    MediaElement.MetaDataTitle="Title";
+    MediaElement.MetaDataArtist="Artist";
+    MediaElement.MetaDataArtworkUrl="http://www.myownpersonaldomain.com/image.jpg";
```

Comment:
I've renamed this class to `Metadata` in the code to use [the correct spelling](https://www.merriam-webster.com/dictionary/metadata)

```suggestion
    MediaElement.MetadataTitle="Title";
    MediaElement.MetadataArtist="Artist";
    MediaElement.MetadataArtworkUrl="http://www.myownpersonaldomain.com/image.jpg";
```